src/commandHandlers/bio.ts

```diff
@@ -63,6 +63,8 @@ export const command = async (arg: string, embed: MessageEmbed, message: Message
             } catch (e) {
                 log.error(`ERROR: Couldn't get location ${data}`);
             }
+        } else if (field === 'twitter') {
```

Comment:
I think an `if` here would be better

Reply:
Please give me insight for ```switch```. We would have to implement a case for each field. Or should we have it so all other fields to fall through? Shouldn't we save the default for logging?

Reply:
Not sure if a default is needed? I was thinking something like this...

```typescript
switch (field) {
   case 'location':
     try {
        // ...
     } catch (e) {
        // ...
     }
   case 'twitter':
      data = `https://twitter.com/${data}`
}
```

What do you think?

Switch docs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch

Reply:
I forgot you don't need the default. I have a habit of making sure I'm using default in a switch statement.

Reply:
I think some linters might complain. Hopefully it won’t and it is an improvement for the future